democracy_club/settings/base.py

```diff
@@ -224,6 +226,7 @@ def blog_markdown(value):
 GOCARDLESS_REDIRECT_URL = "https://democracyclub.org.uk/donate/process/"
 
 SITE_TITLE = "Democracy Club"
+CANONICAL_URL = "https://democracyclub.org.uk"
```

Comment:
Now we have the new deployment you can replace this with `CANONICAL_URL =  f"https://{os.environ.get("FQDN")}"`, as we pass that in to the env on each deployment